.github/resources/.actions/cli-runner/action.yml

```
@@ -0,0 +1,130 @@
+name: Slack CLI Installation and Command Runner 
+description: Download and cache the Slack CLI and run the input command 
+
+inputs:
+  command:
+    description: "Slack CLI command to run"
```

```suggestion
    description: "Slack CLI command to run. Should not include the 'slack' prefix."
```

📚 suggestion: We can document some of these requirements here for the code curious! I'm never sure if these are surfaced elsewhere though...

---

Thanks for this feedback - agreed! 🙌